pinecone/control/models/index_description.py

```diff
@@ -0,0 +1,13 @@
+from typing import NamedTuple
```

Comment:
Pulled this out of legacy `manage.py`. Not sure if we really want to keep it or not long term.